### ISSUE TITLE
RangeError: Invalid time value when using scheduled or deadline as Schedule End

### ISSUE THREAD

**Issue by AngeloChen14** [haydenull/logseq-plugin-agenda#41]
I can set `scheduled`/`deadline` as Schedule Start, but can't set either of them as Schedule End. 

query:

![](https://mdimg-backup.oss-cn-shanghai.aliyuncs.com/2022/04/19/13-18-01-cef27.png)

error msg:

```
Query Exec Error:
calendar: journal
query: [:find (pull ?block [*])
:where
[?block :block/marker ?marker]
[?block :block/scheduled]
[?block :block/deadline]
[(contains? #{"TODO" "DOING" "NOW" "LATER" "WAITING" "DONE"} ?marker)]]

message: Invalid time value
```
console output:
```
RangeError: Invalid time value
    at vDe (file:///home/angelo/.logseq/plugins/logseq-agenda/dist/assets/vendor.2bdf92ec.js:561:65739)
    at file:///home/angelo/.logseq/plugins/logseq-agenda/dist/assets/index.2e0f44d2.js:98:102
    at Array.map (<anonymous>)
    at file:///home/angelo/.logseq/plugins/logseq-agenda/dist/assets/index.2e0f44d2.js:96:814
```

platform: 
Logseq 0.6.6, Arch Linux

**Comment by haydenull** [comment]
May I know what is the version of the agenda plugin?

**Comment by AngeloChen14** [comment]
> May I know what is the version of the agenda plugin?

v1.15.0, download from marketplace

**Comment by haydenull** [comment]
Execute this query in the DevTools and look at what is the output？

https://haydenull.github.io/logseq-plugin-agenda/other/testQuery.html

**Comment by AngeloChen14** [comment]
> Execute this query in the DevTools and look at what is the output？
> 
> https://haydenull.github.io/logseq-plugin-agenda/other/testQuery.html

The output is normal when I click Execute this query in the DevTools in Calendar Queries
![](https://mdimg-backup.oss-cn-shanghai.aliyuncs.com/2022/04/19/23-10-50-9c780.png)

The error message pop up when I click Save in Calendar Settings
![](https://mdimg-backup.oss-cn-shanghai.aliyuncs.com/2022/04/19/23-08-36-e0838.png)

**Comment by haydenull** [comment]
Just released version 1.15.1, you can try again.

**Comment by AngeloChen14** [comment]
Thanks! The problem is fixed now.
